### ISSUE TITLE
How to force valid JWT tokens

### ISSUE THREAD

**Issue by 0x6e6562** [calebmer/postgraphql#62]
I love this project, it is so concise and to the point, especially the scope of what it is trying to achieve!

So I was able to generate a token and have posgraphql evaluate the claim:

```
curl -XPOST -H "Authorization:Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJwb3N0Z3JhcGhxbCIsInJvbGUiOiJncmFwaHFsIiwidXNlcl9pZCI6Mn0.oQVZfTK-LdC1btVvR8OrzRAhSPEKsFJHeqxhF9xk8rc" -H 'Content-Type:application/graphql' -d '{ searchPosts(search:"s",first:2){nodes{rowId}}}' 127.0.0.1:3000
```

So I was able to get it to return valid date for a valid token containing a valid role.

However, I also able to get the same data with an empty token:

```
curl -XPOST -H "Authorization:Bearer " -H 'Content-Type:application/graphql' -d '{ searchPosts(search:"s",first:2){nodes{rowId}}}' 127.0.0.1:3000
```

Is there a way to mandate a non-empty token if the secret flag has been set (other than say enforcing this with an upstream nginx proxy)?

Any other non-empty token string appears to evaluate correctly (it would be nice to get a 403 instead of a 500, but that's a separate issue).

**Comment by calebmer** [comment]
Thanks so much for reporting! Yep, this is definitely a bug because right [here](https://github.com/calebmer/postgraphql/blob/master/src/createServer.js#L81) we just return for invalid `Authorization` headers instead of throwing an error. So a simple bugfix if anyone wants to submit a PR.

Also, returning the correct error code is pretty important. Whoops 😊

**Comment by ferdinandsalis** [comment]
@0x6e6562 if you get it working. I would love to see an authentication example ☺️

**Comment by calebmer** [comment]
Fixed in #72, and I threw in correct HTTP status codes for good measure 😊

**Comment by calebmer** [comment]
Released in `1.6.0` 🎉
